webcompat/templates/new-issue.html

```diff
@@ -25,7 +25,7 @@
 
 {%- block extrascripts -%}
 {% if ab_active('exp') == 'form-v2' %}
-<script src="{{ url_for('static', filename='js/lib/issue-wizard-bugform.js') }}"></script>
+<script type="module" src="{{ url_for('static', filename='js/lib/wizard/app.js') }}"></script>
```

Comment:
Do we have a bug filed for the `nomodule` babel bundle version?

(This makes me think we should possibly be merging into a branch until that's ready (rather than master))

Reply:
I haven't filed it yet, was planning on adding it and build related issues once I sort out the functionality